openrtb2/native.go

```diff
@@ -22,6 +23,23 @@ import (
 // However, any given bid for the impression must conform to one of the offered types.
 type Native struct {
 
+	// Attribute:
+	//   request_native
+	// Type:
+	//   object; required
+	// Description:
+	// 		Native Struct for BidSwitch
+	//		https://protocol.bidswitch.com/ssp-protocol/ssp-native-request.html
+	RequestNative *native1.Request `json:"request_native"`
```

Comment:
Add the omitempty

Reply:
And also Line 60: 
Request string `json:"request"` we need to add the omitempty